### PR TITLE
csranking: kui ren, wenbo shen, jinsong han, Zhejiang University. dbl…

### PR DESCRIPTION
**Pull request by Jeimon**
…p: kui ren and kui ren 0001

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by Jeimon** [comment]
#

File: dblp-aliases.csv [hunk 1]
```diff
@@ -2193,3 +2193,4 @@ Muhammad Zubair Shafiq,Zubair Shafiq
 Oscar Cánovas Reverte,Óscar Cánovas Reverte
 Óscar Cánovas,Óscar Cánovas Reverte
 Oznur Tastan,Öznur Tastan
+Kui Ren,Kui Ren 0001
```
Comment (emeryberger) [inline dblp-aliases.csv:2196]: Please delete this. The main entry should be the DBLP entry (that is, Kui Ren 0001).

File: csrankings.csv [hunk 2]
```diff
@@ -16113,3 +16113,6 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,Norwegian University of Science and Technology,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Kui Ren,Zhejiang University,https://person.zju.edu.cn/kuiren,uuQA_rcAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:16116]: Use the DBLP name (Kui Ren 0001).

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Closing for lack of response in > 2 weeks.

**Pull request closed without merging**